shared/teams/team/member/index.new.tsx

```diff
@@ -176,7 +256,10 @@ const SubteamInRow = (props: SubteamInRowProps) => {
         <Kb.Avatar teamname={props.subteam.teamname} size={32} />
         <Kb.Box2 direction="vertical" alignItems="flex-start">
           <Kb.Text type="BodySemibold">{props.subteam.teamname}</Kb.Text>
-          <Kb.Text type="BodySmall">Joined Feb 1944 {/* TODO: where to get this data? */}</Kb.Text>
+          <Kb.Text type="BodySmall">
+            Joined{' '}
+            {props.membership.joinTime ? formatTimeForTeamMember(props.membership.joinTime) : 'this team'}
```

Comment:
maybe instead of `Joined this team` just omit the row if we don't have the data? It's clever but IMO just more confusing than leaving it empty.

Reply:
i just used it for storybook work :) sure!